pkg/nsx/services/common/types.go

```diff
@@ -34,11 +34,18 @@ const (
 	TagScopeNCPVNETInterface        string = "ncp/vnet_interface"
 	TagScopeVPCCRName               string = "nsx-op/vpc_cr_name"
 	TagScopeVPCCRUID                string = "nsx-op/vpc_cr_uid"
+	// TagScopeSubnetCRType indicates that NSX Subnet is linked to Subnet CR or SubnetSet CR.
+	TagScopeSubnetCRType string = "nsx-op/subnet_cr_type"
+	TagScopeSubnetCRUID  string = "nsx-op/subnet_cr_uid"
 
-	GCInterval    = 60 * time.Second
-	FinalizerName = "securitypolicy.nsx.vmware.com/finalizer"
+	LabelDefaultSubnetSet    string = "nsxoperator.vmware.com/default-subnetset-for"
```

Comment:
how about DefaultSubnetSetLabelKey, DefaultPodSubnetSetLabelValue and DefaultVMSubnetSetLabelValue?

Reply:
@heypnus 's patch had already added this label, and I have rebased her patch.